UWPCommunity/App.xaml.cs

```diff
@@ -192,17 +192,10 @@ private void ExtendIntoTitleBar()
 
                 ApplicationView.GetForCurrentView().TitleBar.ButtonForegroundColor = (Color)Current.Resources["SystemAccentColor"];
                 ApplicationView.GetForCurrentView().TitleBar.ButtonBackgroundColor = Colors.Transparent;
-                ApplicationView.GetForCurrentView().TitleBar.ButtonHoverForegroundColor = (Color)Current.Resources["SystemAccentColor"];
-                ApplicationView.GetForCurrentView().TitleBar.ButtonHoverBackgroundColor = Colors.Transparent;
             }
             else
             {
                 coreTitleBar.ExtendViewIntoTitleBar = false;
-
```

Comment:
These lines shouldn't be removed. They reset the button colors so that when the user disables extending into the title bar, they don't get this weirdness:
![image](https://user-images.githubusercontent.com/7549325/87975519-930dc800-ca80-11ea-8fd6-b77b0ba57440.png)

Reply:
I am not sure what behavior you are referring to here. I've merged with master (as there have been a lot of changes in the last two weeks), and the app seems to behave correctly.

Reply:
1.2.0 introduces a new setting that disables extending into the title bar. Those null assignments reverts the colors so it appears properly in the unmodified, not-extended-into title bar.

Reply:
I've tested both switching in "Extend View into titlebar mode" and switching in non "extend view into titlebar mode", and both look fine with this PR.

Reply:
Weird... which version of Windows are you running?

Reply:
Insider Build 20170. What steps did you run to get that screenshot? Was that also reproducable with the latest merge with master?

Reply:
I'm on 19041. All I did was remove those null setters and switch the "Extend view into title bar" option on and off

Reply:
Since I can't repro this on my machine, I've readded the two null setter in question.

Reply:
Alright then, maybe this is something that should be moved into it's own issue

Reply:
Maybe this is some oddity with the insider build, if it works correctly on released builds, I think that's fine.